server/src/test/java/org/spine3/server/validate/MessageValidatorShould.java

```diff
@@ -565,21 +582,9 @@ private void digitsCountTest(double value, boolean isValid) {
     }
 
     private void assertMessageIsValid(boolean isValid) {
-        assertMessageIsValid(isValid, validator);
-    }
-
-    private void assertCmdMessageIsValid(boolean isValid) {
-        assertMessageIsValid(isValid, cmdValidator);
-    }
-
-    private static void assertMessageIsValid(boolean isValid, MessageValidator validator) {
-        if (isValid) {
-            assertFalse(validator.isMessageInvalid());
-            assertTrue(validator.getErrorMessage().isEmpty());
-        } else {
-            assertTrue(validator.isMessageInvalid());
-            assertFalse(validator.getErrorMessage().isEmpty());
-        }
+        assertEquals(isValid, validator.isMessageValid());
+        assertEquals(!isValid, validator.isMessageInvalid());
```

Comment:
As long the class is named `MessageValidator`, we can have just `isValid()` to make it shorter.